INSTALL.md

```diff
@@ -419,6 +420,60 @@ so, you will need to edit `homeserver.yaml`, as follows:
 For a more detailed guide to configuring your server for federation, see
 [federate.md](docs/federate.md).
 
+## Client Well-known URI
+
+Setting up the client Well-Known URI is optional but if you set it up, it will
+allow users to enter their full username (e.g. `@user:server.com`) into clients
```

Comment:
It would be nice if this example and below were consistent (e.g. I believe `server.com` and `<server_name>` should match?)

Reply:
Yeah I just wasn’t sure how closely to match them. This is more like an example of a valid username (to compare to most people thinking their username is just `user` or `@user`) whereas below is more like, fill this in with your server name, but maybe it would make sense to match them up.

Reply:
I think making it as consistent as possible is good, but it isn't a huge deal.

Reply:
Ok I changed this to `@user:<server_name>`